viz_config.yml

```diff
@@ -0,0 +1 @@
+storm_code: al152017 # find the right code at https://www.nhc.noaa.gov/gis/archive_besttrack.php?year=2018
```

Comment:
Looks fine to me. In the past we did split this as "ocean" "stormnum" and "year" (or something like that). I don't really see any reason we need to split them...just noting the difference and this seems :+1: to me.

Reply:
yeah, i saw that and decided to merge the three parameters because i also couldn't see a big need to split them. it'd be one thing if the user already knew the answer to all three params independently and just had to pass them in to get the correctly formatted code, but because somebody will always have to go to the NHC website to find out which storm number it is, i think it makes sense to copy-paste the entire 8-character code at that point.